### PR TITLE
[SELC-4736] Feat: Added script for institutionDescription migration from selcOnboarding.Onboarding to selcMsCore.Institution

### PR DESCRIPTION
**Pull request by flaminiaScarciofolo**
#### List of Changes
- Added script for institutionDescription migration from selcOnboarding.Onboarding to selcMsCore.Institution

#### Motivation and Context
Some Institution documents don't have institutionDescription so we have to migrate this field from onboarding collection of selcOnboarding DB

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/user-core-migration/query.py [hunk 1]
```diff
@@ -22,11 +22,45 @@ def get_institutions(page, size):
         *paging(page, size)
     ]
 
-def get_institution(institutionId):
+
+def get_institutions_from_onboardings(page, size):
```
Comment (manuraf) [inline apps/user-core-migration/query.py:26]: you could filter just for status COMPLETED

File: apps/user-core-migration/query.py [hunk 2]
```diff
@@ -22,11 +22,45 @@ def get_institutions(page, size):
...
+                'description': {
+                    '$first': '$institution.description'
+                }
+            }
+        },
+        *paging(page, size)
+    ]
+
+
+def count_institutions_from_onboardings():
```
Comment (manuraf) [inline apps/user-core-migration/query.py:40]: ditto

File: apps/user-core-migration/migrate_institution_description.py [hunk 3]
```diff
@@ -16,12 +16,55 @@
...
+        for page in range(START_PAGE, pages):
+            print("Start page " + str(page + 1) + "/" + str(pages))
+
+            onboarding_pages = client[ONBOARDING_DB][ONBOARDINGS_COLLECTION].aggregate(
+                get_institutions_from_onboardings(page, BATCH_SIZE)
+            )
+
+            for institution in onboarding_pages:
+                institution_desc = institution.get('description')
+                institution_tax_code = institution.get('_id')
```
Comment (manuraf) [inline apps/user-core-migration/migrate_institution_description.py:42]: should it be institution_id?

File: apps/user-core-migration/migrate_institution_description.py [hunk 4]
```diff
@@ -16,12 +16,55 @@
...
+
+                client[CORE_DB][INSTITUTION_COLLECTION].update_many(
+                    {"taxCode": institution_tax_code},
```
Comment (manuraf) [inline apps/user-core-migration/migrate_institution_description.py:45]: in case here you should use "_id" as filter

**Review by manuraf — CHANGES_REQUESTED** [review]

File: apps/user-core-migration/migrate_institution_description.py (hunk 3 above)
Comment (flaminiaScarciofolo) [reply apps/user-core-migration/migrate_institution_description.py:42]: _id field is returned from groupBy operation and query group by taxCode in onboarding collection.

**Review by manuraf — APPROVED** [review]

**Pull request merged**